packages/colorpicker/README.md

```diff
@@ -0,0 +1,29 @@
+# @zendeskgarden/react-colorpicker [![npm version](https://flat.badgen.net/npm/v/@zendeskgarden/react-colorpicker)](https://www.npmjs.com/package/@zendeskgarden/react-colorpicker)
+
+This package includes components related to colorpicker in the
+[Garden Design System](https://zendeskgarden.github.io/).
+
+## Installation
+
+```sh
+npm install @zendeskgarden/react-colorpicker
+
+# Peer Dependencies - Also Required
+npm install react react-dom prop-types styled-components @zendeskgarden/react-theming
+```
+
+## Usage
+
+```jsx
+import { ThemeProvider } from '@zendeskgarden/react-theming';
+import { ColorPicker } from '@zendeskgarden/react-colorpicker';
+
+const [color, setColor] = React.useState('#17494D');
```

Comment:
Should we include a wrapping `Example` function around this to make it a valid example? Take a look at `Tabs/README.md`

Reply:
Ah yes, I don't want a floating `useState` in the example. It's actually not necessary, so I've removed it and simplified the example code.